lib/ansible/modules/cloud/azure/azure_rm_networkinterface.py

```diff
@@ -179,6 +179,23 @@
             - When a default security group is created for a Linux host a rule will be added allowing inbound TCP
               connections to the default SSH port 22, and for a Windows host rules will be added allowing inbound
               access to RDP ports 3389 and 5986. Override the default ports by providing a list of open ports.
+    enable_accelerated_networking:
+        description:
+            - This enables accelerated networking on the network interface
+        aliases:
```

Comment:
no need alias for newly added parameters

Reply:
I thought this could be used as a convenience thing, since the API parameter name is so long; I will remove it.